cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -251,7 +251,7 @@ def continue_cherry_pick(self):
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.version_option(version="0.0.2.post2")
+@click.version_option(version="0.1.0.dev1")
```

Comment:
🤔 When I removed __init__.py, getting this error message during `flit install`:
```
FileNotFoundError: [Errno 2] No such file or directory: 'cherry_picker/__init__.py'
```
So I kept the file..